cal/cal/static/css/calendaranalytics.scss

```diff
@@ -25,6 +25,12 @@ button {
   padding: 0 5 5 5;
 }
 
+.textbox input {
```

Comment:
Is there a particular reason why you want it on `input` instead of `.textbox input`? When I change it to `input`, it puts an extra line in between the checkbox and the label name, which doesn't look particularly good. I'll also change the naming.